pom.xml

```diff
@@ -104,7 +104,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>git</artifactId>
-            <version>2.0</version>
+            <version>4.0.0-rc</version>
```

Comment:
should we really us 4-RC (not stable) when original issue says about  v3?

Reply:
Hi,
In issue we have a little mismatch. In the same time was release two plugins:
[git-client](https://plugins.jenkins.io/git-client) 3.0.0-rc version
[git](https://plugins.jenkins.io/git) 4.0.0-rc version

Original issue say about git-client version but this plugin use git plugin.
Unfortunately RC versions was deployed to public Jenkins repository and any client can download that versions. I think if any plugins are deployed to public repository is a stable ;)

Reply:
You're absolutely right, but all user have git plugin and git-client in their Jenkins. Many of them executed jenkins update automatically and got a new versions git and git-client. In that situation stashnotifier plugin doesn't work.

Reply:
Sounds legit